docs/modules/angular/pages/ngrx/ngrx.adoc

```diff
@@ -39,12 +39,13 @@ NOTE: Folders should only be created and used if they have content.
 │   │   │    │    │    ├── featurecomponent1.component.ts
 │   │   │    ├── directives     // optional - directives used in specific feature module
 │   │   │    ├── pages
-│   │   │    │    ├── page1       // one of pages, might have many of them
-│   │   │    │    │     ├── page1subcomponent1    // one of subcomponent for page, optional + might have many of them
+│   │   │    │    ├── page1       // each pages gets a subfolder, might have many of them
+│   │   │    │    │     ├── page1subcomponent1    // one of subcomponent for page, optional + might have many of them, this can also be a dialog that is only used by this pages
```

Comment:
dialogs should be in a subfolder (page1/dialogs/dialog1...) to better structure the files